src/fides/core/dataset.py

```diff
@@ -30,7 +31,7 @@
     "postgresql": ["information_schema"],
     "mysql": ["mysql", "performance_schema", "sys", "information_schema"],
     "mssql": ["INFORMATION_SCHEMA", "guest", "sys"],
-    "snowflake": ["information_schema"],
+    "snowflake": ["INFORMATION_SCHEMA"],
```

Comment:
Is there any method behind the case inconsistency of these different datastores you're aware of?

Reply:
Snowflake just happens to default to all caps, it's a common gotcha when people move to Snowflake and are used to using something like `CamelCase`. Some users will initially try and double-quote columns to maintain casing across data stores because it feels a bit funky that any non-quoted item defaults to all caps. MSSQL is case sensitive by default and just happens to implement the `INFORMATION_SCHEMA` in that case. It is sort of a window into the wild west that can be different databases and their implementation of SQL :)